wagtailio/core/blocks.py

```diff
@@ -583,6 +587,26 @@ class Meta:
         template = "patterns/components/get-started/get-started.html"
 
 
+class LoopingVideoBlock(blocks.StructBlock):
+    loop = blocks.BooleanBlock(required=False, default=True)
+    autoplay = blocks.BooleanBlock(required=False, default=False)
```

Comment:
discussed: leave only autoplay (and maybe with a note that it autoplays and loops)

Reply:
thanks, updated in 34198484